### ISSUE TITLE
Creating srt translation pairs from Garchen Rinpoche Catalog with interpreter in Audio files.

### ISSUE THREAD

**Issue by gangagyatso4364** [OpenPecha/garchen-rinpoche-speech-sig#50]
### Description
We currently have garchen rinpoche catalog with audio files and metadata about the existence of interpreter in audio files or not. 
What we need to from audio files that has interpreter in it is to generate an srt file that has label for speakers in the audio. The garchen rinpoche speaking instances will be labeled GR and Interpreter speaking instances will be labeled Interpreter. Also we will generate transcript for GR speaking instances transcript using finetuned garchen rinpoche model and for other speakers we will use whisper large v3 model to generate transcript. along with interpreter language label. 
Also we will create srt files with only interpreter transcripts. and another with only garchen rinpoche transcript. 
Once we have generated srt files with interpreters. we will update catalog with link to srt files. 

### Completion Criteria
A srt files with both garchen rinpoche transcript and interpreter transcript transcribed. 

### Subtasks
- [x] list down audios files with interpreter and path to thier speaker diarisation files 
- [x] run inference on audio files with interpreter using whisper large v3  and finetuned model for gr segments.
- [x] update catalog with link to srt files.
- [x] document catalog columns

reviewer 
- [x] @dhakar66

**Comment by tenkus47** [comment]
there is some update needed on the card, @gangagyatso4364  wll update the card accordingly.

**Comment by gangagyatso4364** [comment]
https://github.com/vasistalodagala/whisper-finetune?tab=readme-ov-file#hyperparameter-tuning
